core/unit_test/TestRangePolicyConstructors.hpp

```diff
@@ -204,8 +204,6 @@ constexpr bool test_chunk_size_explicit() {
 #ifdef KOKKOS_ENABLE_DEPRECATED_CODE_4
   static_assert(std::is_convertible_v<int, ChunkSize>);
   static_assert(std::is_constructible_v<ChunkSize, int>);
-  // FIXME some execution spaces are implicitly constructible from int
-  // which makes the constructor call ambiguous.
```

Comment:
I would keep the comment without the FIXME.
```suggestion
  // Some execution spaces were implicitly constructible from int
  // which made the constructor call ambiguous.
```

Reply:
Agreed please keep the comment